dvl_gazebo/models/teledyne_whn/model.sdf

```diff
@@ -0,0 +1,369 @@
+<?xml version="1.0" ?>
+<!-- NOTE: The SDF version of this model is untested due to the ROS
+     robot_state_publisher's inability to work with SDF models.  It may work
+     when mounted to a robot with plugins that broadcast their transforms. -->
+<sdf version="1.6">
+  <model name="teledyne_whn">
+    <link name="dvl_base_link">
+      <inertial>
+        <pose>0 0 0 0 0 0</pose>
+        <mass>0.001</mass>
+        <inertia>
+          <ixx>0.000000017</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.000000017</iyy>
+          <iyz>0</iyz>
+          <izz>0.000000017</izz>
+        </inertia>
+      </inertial>
+      <visual name="visual">
+        <pose>0 0 0 0 0 0</pose>
+        <geometry>
+          <mesh>
+            <uri>model://teledyne_whn/meshes/teledyne_whn.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <collision name="collision">
+        <pose>0 0 0 0 0 0</pose>
+        <geometry>
+          <cylinder>
+            <radius>0.000001</radius>
```

Comment:
Would be better to have the collision dimensions be roughly the same as the DVL housing.

Reply:
Good catch.  I did that in the urdf version, but didn't clean up the final SDF version (which I can't test without adding it to a robot with a plugin that's broadcasting the transforms).